handlers/ocr-handler/README.md

```diff
@@ -0,0 +1,8 @@
+# OCR Handler
+## What is OCR?
+OCR is an acronym for Optical Character Recognition which is the process of extracting machine-readable text pictured in graphics or documents.
+## About this handler
+The OCR handler takes data output from our [OCR preprocessor](https://github.com/Shared-Reality-Lab/IMAGE-server/tree/main/preprocessors/ocr) 
+as well as other preprocessors that provide data regarding objects or segments in a graphic. The handler reconciles the bounding boxes of the discovered
```

Comment:
You may want to link to these as well or specify the data types that are compatible.